torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -571,6 +574,56 @@ def test_backend_group(self):
         def test_backend_full_group(self):
             self._test_group_override_backend(self._init_full_group_test)
 
+        # NCCL Batch SEND RECV
+        @skip_if_no_gpu
+        @unittest.skipIf(BACKEND != "nccl", "NCCL Send Recv Only")
```

Comment:
Doesn't this work for Gloo as well?

Reply:
Based on https://pytorch.org/docs/stable/distributed.html, seems that we haven't officially claimed GLOO supports send/recv on GPUs. So I just tested NCCL send/recv here.